backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/json/Table.java

```diff
@@ -209,4 +211,12 @@ public String getSchemaId() {
   public void setSchemaId(String schemaId) {
     this.schemaId = schemaId;
   }
+
+  public String[] getProfiles() {
+    return profiles;
```

Comment:
returns null is set to null

Reply:
when empty should be null indeed. Is that would you mean?